File: StxInsider/app/src/main/java/com/stxnext/stxinsider/viewmodel/ViewWrapper.java
package com.stxnext.stxinsider.viewmodel;

import android.support.v7.widget.RecyclerView;
import android.view.View;

/**
 * Created by bkosarzycki on 12/29/15.
 */
public class ViewWrapper<V extends View> extends RecyclerView.ViewHolder {

    private V view;

    public ViewWrapper(V itemView) {
        super(itemView);
        view = itemView;
    }

    public V getView() {
        return view;
    }
}

File: StxInsider/app/src/main/java/com/stxnext/stxinsider/viewmodel/ViewWrapper.kt
package com.stxnext.stxinsider.viewmodel

import android.support.v7.widget.RecyclerView
import android.view.View

/**
 * Created by bkosarzycki on 12/29/15.
 */
class ViewWrapper<V : View>(val view: V) : RecyclerView.ViewHolder(view)
